packages/botonic-nlp/src/tasks/text-classification/botonic-text-classifier.ts

```diff
@@ -0,0 +1,153 @@
+import { join } from 'path'
+
+import { DatasetLoader } from '../../dataset/loader'
+import { Dataset, Sample } from '../../dataset/types'
+import { generateEmbeddingsMatrix } from '../../embeddings/embeddings-matrix'
+import { WordEmbeddingStorage } from '../../embeddings/types'
+import { ModelManager } from '../../model/manager'
+import { ModelEvaluation } from '../../model/types'
+import { PADDING_TOKEN, UNKNOWN_TOKEN } from '../../preprocess/constants'
+import { Preprocessor } from '../../preprocess/preprocessor'
+import { trainTestSplit } from '../../preprocess/selection'
```

Comment:
what about grouping imports within a same module?
e.g.:
```
// model/index.ts
export * from './manager'
export * from './types'
```
then here:
```
import {ModelManager, ModelEvaluation} from './model'
```

Reply:
Makes sense! I opened a [ticket](https://linear.app/hubtype/issue/BOT-326/group-imports-of-nlp-tasks) for this.